### DISCUSSION TITLE
How i can retrieve geometry column without including the CRS object.

### DISCUSSION THREAD

**Discussion by mudassertufail** [sequelize/sequelize#16131] [Help & Questions]
I am using the PostGIS geometry in the Postgre db in my application. I am using sequelize ORM to get the data from the db. When I get the geometry field data it returns the data like mentioned below:

“location”: {
                    “crs”: {
                        “type”: “name”,
                        “properties”: {
                            “name”: “EPSG:4326”
                        }
                    },
                    “type”: “Point”,
                    “coordinates”: [
                        31.0120136,
                        -155.3959561
                    ]
In the response, I am getting the CRS along with the geometry data. I do not need this CRS. I need only the geometry data like below

“location”: [
                    “type”: “Point”,
                    “coordinates”: [
                        31.0120136,
                        -155.3959561
                    ]
How I can retrieve the geometry column without including the CRS object using the sequlize in my node app?

**Comment by ephys** [comment]
You'd need to use raw sql to convert the geometry to JSON (I think `ST_AsGeoJSON`), then use postgres's [JSON functions](https://www.postgresql.org/docs/current/functions-json.html) to remove the properties you do not want, then convert it back using `ST_GeomFromGeoJSON`.
